lib/mayaUsd/ufe/UsdSceneItemOps.cpp

```diff
@@ -60,14 +75,24 @@ Ufe::SceneItem::Ptr UsdSceneItemOps::sceneItem() const
 
 Ufe::UndoableCommand::Ptr UsdSceneItemOps::deleteItemCmd()
 {
-	auto deleteCmd = UsdUndoDeleteCommand::create(prim());
-	deleteCmd->execute();
-	return deleteCmd;
+	if (prim().IsActive()) {
+		auto deleteCmd = UsdUndoDeleteCommand::create(prim());
+		deleteCmd->execute();
+		return deleteCmd;
+	}
+
+	displayWarning(prim(), kWarningCannotDeactivePrim);
+	return nullptr;
```

Comment:
As per spec - if user tries to delete an inactive prim, print a warning message and don't execute command.